woocommerce/wc-admin
# Add experimental list documentation

This PR aims to add documentation to the [experimental list](https://github.com/woocommerce/woocommerce-admin/tree/main/packages/experimental/src/experimental-list).

After adding a few changes to the `experimental-list` such as [this](https://github.com/woocommerce/woocommerce-admin/pull/7632) and [this](https://github.com/woocommerce/woocommerce-admin/pull/7611) there are new behaviors and features that are not documented so I added a README.md and a storybook to show how to use it.

(No changelog needed).

### Screenshots
![screenshot-localhost_6007-2021 09 21-11_39_46](https://user-images.githubusercontent.com/1314156/134192004-a549e45e-11e9-433f-aad9-b6566d4e3c26.png)


### Detailed test instructions:

-   Verify the README.me looks ok.
-   Run `npm run storybook`.
-   In the Storybook, go to `Experimental` > `List` > `TaskItem`.
-   Verify that everything looks good.
